crates/store/re_protos/proto/rerun/v0/remote_store.proto

```diff
@@ -67,6 +71,18 @@ message TimeMetadata {
     TimeRange time_range = 2;
 }
 
+// ---------------- UpdateRecordingMetadata  -----------------
+
+message UpdateRecordingMetadataRequest {
+    RecordingId recording_id = 1;
+    RecordingMetadata metadata = 2;
+}
+
+message UpdateRecordingMetadataResponse {
+    RecordingId id = 1;
+    RecordingMetadata metadata = 2;
```

Comment:
Is there a reason we need to return this? Seems like all we should need here is a success/fail.

Reply:
I was kinda looking at parity with register, but indeed that has different semantics, so it can be an empty response. I thought it might be useful for a user to now again see the fully updated recording, but it you think there's no value - easy to drop!